modules/schema/src/test/java/org/apache/ignite/internal/schema/configuration/KnownDataStorageValidatorTest.java

```diff
@@ -39,6 +40,7 @@
 @ExtendWith(ConfigurationExtension.class)
 public class KnownDataStorageValidatorTest {
     @InjectConfiguration(
+            value = "mock.name = " + UnknownDataStorageConfigurationSchema.UNKNOWN_DATA_STORAGE,
```

Comment:
Oh, so you didn't remove the unknown storage. Why?

Reply:
The same https://github.com/apache/ignite-3/pull/1872#discussion_r1165974751

Reply:
Removed